helm_deploy/hmpps-staff-lookup-service/values.yaml

```diff
@@ -43,3 +48,4 @@ generic-service:
 
 generic-prometheus-alerts:
   targetApplication: hmpps-staff-lookup-service
+  alertSeverity: hmpps-tier
```

Comment:
Hmm.. we probably need another slack channel? I don't know..

Reply:
this is not named the greatest, this means go to slack channel manage_a_workforce_dev

Reply:
It's just a mapping to our channel